lib/cosmos/tools/limits_monitor/limits_monitor.rb

```diff
@@ -343,10 +402,23 @@ def limits_change(target_name, packet_name, item_name, state)
 
     # Record the stale packet and generate a log message
     def stale_packet(target_name, packet_name)
-      out_of_limit([target_name, packet_name, nil])
+      item = [target_name, packet_name, nil]
+      unless (@stale.includes_item?(item) || @ignored_stale.includes_item?(item) || UNKNOWN_ARRAY.includes_item?(item))
+        @stale << item
+        @items["#{item[0]} #{item[1]}"] = @new_item_callback.call(*item) 
+      end
       return ["INFO: Packet #{target_name} #{packet_name} is STALE\n", :BLACK]
     end
 
+    def refresh_stale_packet(target_name, packet_name)
+      item = [target_name, packet_name, nil]
+      index = @stale.delete_item(item)
+      widget = @items.delete("#{target_name} #{packet_name}") if index
+      @remove_item_callback.call(widget) if widget
+      # TBD get all out-of-limits items for the packet that is no longer stale.
```

Comment:
I'm not sure this TBD is necessary.  Won't the CmdTlmServer notify you of everything that is out of limits?